spanner/api/Program.cs

```diff
@@ -928,34 +1091,55 @@ internal class CustomTransientErrorDetectionStrategy
             return ExitCode.Success;
         }
 
-        public static object QueryDataWithTransaction(string projectId,
-            string instanceId, string databaseId)
+        public static object QueryDataWithTransaction(string projectId, string instanceId, string databaseId, string platform)

```

Comment:
Code lines within doc tags [START ...] [END ...] should be less than 80 characters long.  There are other lines in this PR that are > 80 chars too.